examples/auctions/blind_auction.vy

```diff
@@ -74,17 +74,17 @@ def bid(_blindedBid: bytes32):
 
 # Returns a boolean value, `True` if bid placed successfully, `False` otherwise.
 @private
-def placeBid(bidder: address, value: uint256) -> bool:
+def placeBid(bidder: address, _value: uint256) -> bool:
```

Comment:
Does this fail with the changes in this PR?

Regardless, could probably move this to a general cleanup PR